src/components/modal/Modal.scss

```diff
@@ -133,3 +135,33 @@
     z-index: 89;
     display: flex;
 }
+
+//Responsive support
+.is-responsive-web {
+    .modal {
+        @include breakpoint($modal-fullscreen) {
+            padding: 0;
+        }
+    }
+
+    .modal-dialog-container {
+        width: 100%;
+        margin: auto;
```

Comment:
Is this needed if it's already declared in the base styles?

Reply:
No we don't. Removed.